test/controllers/api/v2/hosts_bulk_actions_controller_test.rb

```diff
@@ -380,7 +381,7 @@ def test_get_host_traces
       Katello::HostTracer.create(host_id: @host1.id, application: 'kernel', app_type: 'static', helper: 'reboot the system')
       Katello::HostTracer.create(host_id: @host2.id, application: 'dbus', app_type: 'static', helper: 'reboot the system')
 
-      post :traces, params: {
+      get :traces, params: {
```

Comment:
afaik this method is still a POST so I'm curious how this actually works! Can you see what's going on here?

Reply:
Updated to POST to match the bulk controller, here are the results from both:

With POST: 
```ruby
[1] pry(#<Katello::Api::V2::HostsBulkActionsControllerTest>)> response.body
=> "{\"total\":2,\"subtotal\":2,\"page\":1,\"per_page\":20,\"error\":null,\"search\":null,\"sort\":{\"by\":\"application\",\"order\":\"desc\"},\"results\":[{\"id\":890,\"application\":\"kernel\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980191011,\"host\":\"host49\",\"reboot_required\":true},{\"id\":891,\"application\":\"dbus\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980191012,\"host\":\"host50\",\"reboot_required\":true}]}\n"
[2] pry(#<Katello::Api::V2::HostsBulkActionsControllerTest>)> response.body.class
=> String

[1] pry(#<Katello::Api::V2::HostsBulkActionsController>)> collection
=> {:results=>
  [#<Katello::HostTracer:0x0000000018b61238 id: 900, host_id: 980191001, application: "kernel", helper: "reboot the system", app_type: "static">,
   #<Katello::HostTracer:0x0000000018b61080 id: 901, host_id: 980191002, application: "dbus", helper: "reboot the system", app_type: "static">],
 :subtotal=>2,
 :total=>2,
 :page=>1,
 :per_page=>20,
 :error=>nil}
[2] pry(#<Katello::Api::V2::HostsBulkActionsController>)> foo = respond_for_index(:collection => collection, :template => '../../../api/v2/host_tracer/index')
=> "{\"total\":2,\"subtotal\":2,\"page\":1,\"per_page\":20,\"error\":null,\"search\":null,\"sort\":{\"by\":\"application\",\"order\":\"desc\"},\"results\":[{\"id\":900,\"application\":\"kernel\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980191001,\"host\":\"host39\",\"reboot_required\":true},{\"id\":901,\"application\":\"dbus\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980191002,\"host\":\"host40\",\"reboot_required\":true}]}\n"
[3] pry(#<Katello::Api::V2::HostsBulkActionsController>)> foo.class
=> ActionView::OutputBuffer
```

With GET:

```ruby
[1] pry(#<Katello::Api::V2::HostsBulkActionsControllerTest>)> response.body
=> "{\"total\":2,\"subtotal\":2,\"page\":1,\"per_page\":20,\"error\":null,\"search\":null,\"sort\":{\"by\":\"application\",\"order\":\"desc\"},\"results\":[{\"id\":893,\"application\":\"kernel\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980191013,\"host\":\"host51\",\"reboot_required\":true},{\"id\":894,\"application\":\"dbus\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980191014,\"host\":\"host52\",\"reboot_required\":true}]}\n"
[2] pry(#<Katello::Api::V2::HostsBulkActionsControllerTest>)> response.class
=> ActionDispatch::TestResponse

[1] pry(#<Katello::Api::V2::HostsBulkActionsController>)> collection
=> {:results=>
  [#<Katello::HostTracer:0x00000000184b3378 id: 897, host_id: 980190995, application: "kernel", helper: "reboot the system", app_type: "static">,
   #<Katello::HostTracer:0x00000000184b31c0 id: 898, host_id: 980190996, application: "dbus", helper: "reboot the system", app_type: "static">],
 :subtotal=>2,
 :total=>2,
 :page=>1,
 :per_page=>20,
 :error=>nil}
[2] pry(#<Katello::Api::V2::HostsBulkActionsController>)> foo = respond_for_index(:collection => collection, :template => '../../../api/v2/host_tracer/index')
=> "{\"total\":2,\"subtotal\":2,\"page\":1,\"per_page\":20,\"error\":null,\"search\":null,\"sort\":{\"by\":\"application\",\"order\":\"desc\"},\"results\":[{\"id\":897,\"application\":\"kernel\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980190995,\"host\":\"host33\",\"reboot_required\":true},{\"id\":898,\"application\":\"dbus\",\"helper\":\"reboot the system\",\"app_type\":\"static\",\"host_id\":980190996,\"host\":\"host34\",\"reboot_required\":true}]}\n"
[3] pry(#<Katello::Api::V2::HostsBulkActionsController>)> foo.class
=> ActionView::OutputBuffer
```